### DISCUSSION TITLE
Basic Elfin EW11 Unavailable issue (H1 G2 inverter)

### DISCUSSION THREAD

**Discussion by 60milesmile** [nathanmarlor/foxess_modbus#700] [Q&A] [answered]
Evening all - thanks firstly for a great tool.

I've setup a Elfin EW11 and the modbus integration into a vanilla HA install. The Elfin chirps away and has a reserved IP address. I've made up the connections at the base of my H1 inverter (again vanilla setup) and put 5V into the requisite pins (note not from the USB under the inverter as this only hits about 4.7v and meant the Elfin failed to start). 

I've setup the modbus integration and it pulls though the correct entity - however, while it starts as "connected" and a couple of values get pulled though, the connection fails after a short time. I've quadrupple checked the pins and swapped out the original cat5 for some thinker cable, and even moved the whole setup closer to the inverter in case single strength was the issue - but no dice. The current clamp wires (7 and 8) are absolutely fine (and show up in the foxx app with values).

I've also checked and tested the polarity on the data pins, but when put in reverse the modbus integration fails to load the entity at all (so I know this is right way round).

its so frustrating seeing all the entities in HA without ever being populated - any ideas? could it be a duff Elfin, or (worse) lose connections inside the inverter? Its H1-3.7-E-G2, software version 1.14.

(I've restarted HA a few times but not removed and reinstalled the HACs integration - as assume this is not related?)

any tips/support very gratefully welcome. 

Al in london, fustrated.

**Comment by FozzieUK** [comment, marked as answer]
Hi, yes I see you've seen the answer on another post - your firmware is old and Fox changed the register map in the G2 early on.

Once they have updated your Master/Manager versions the integration should start up no problem.
